<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerController</class>
 <widget class="QWidget" name="PlayerController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>628</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>628</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PlayerController</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>604</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="FilePath">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="OpenFile">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>OpenFile</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="CreateSession">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>CreateSession</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="TryBitStreamDemux">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TryBitStreamDemux</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="StartSendData">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>StartSendData</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="DeleteSession">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>DeleteSession</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="TryNetStreamDemux">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TryNetStreamDemux</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="StopSendData">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>StopSendData</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="GetCacheFree">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>GetCacheFree</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="BeginDecode">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>BeginDecode</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="EndDecode">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>EndDecode</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
